package.json

```diff
@@ -14,7 +14,9 @@
     "check": "npm run lint && npm run test:all",
     "build": "babel src --out-dir build",
     "test:watch": "mocha --compilers js:babel/register --recursive src/**/__tests__/*.js --watch",
-    "test:all": "npm run react:13 && npm test && npm run react:14 && npm test",
+    "test:describeWithDOMOnly": "mocha --compilers js:babel/register --recursive src/**/__tests__/describeWithDOM/describeWithDOMOnly-spec.js",
+    "test:describeWithDOMSkip": "mocha --compilers js:babel/register --recursive src/**/__tests__/describeWithDOM/describeWithDOMSkip-spec.js",
+    "test:all": "npm run react:13 && npm test && npm run test:describeWithDOMOnly && npm run test:describeWithDOMSkip && npm run react:14 && npm test && npm run test:describeWithDOMOnly && npm run test:describeWithDOMSkip",
```

Comment:
We didn't technically need it for .skip, but I felt like having it be consistent would be the way to go.  We do need a separate run for the .only tests though, otherwise it will skip the rest of the test suite.  Happy to change things up a bit to be less verbose if you have an idea though.